NPE in 2023.1 EAP 4 (231.6890.12)

```
IntelliJ IDEA with Maho 2023.1 EAP (Community Edition)
Build #IC-231.6890.12, built on February 9, 2023
Runtime version: 17.0.4+8-LTS amd64
VM: OpenJDK 64-Bit Server VM by SAP SE
Windows 10 10.0
GC: G1 Young Generation, G1 Old Generation
Memory: 8192M
Cores: 32
Non-Bundled Plugins:
    io.acari.DDLCTheme (88.1-1.4.0)
    amadeus.maho (231.6890.12)

Kotlin: 231-1.8.0-release-345-IJ6890.12
```
```
#c.i.o.o.ConfigurableEP - Cannot create configurable
com.intellij.diagnostic.PluginException: Cannot create class io.unthrottled.doki.settings.ThemeSettingsUI (classloader=PluginClassLoader(plugin=PluginDescriptor(name=The Doki Theme, id=io.acari.DDLCTheme, descriptorPath=plugin.xml, path=D:\Project\AmadeusSystems\amadeus.maho.intellij\intellij.run\plugins\doki-theme-jetbrains, version=88.1-1.4.0, package=null, isBundled=false), packagePrefix=null, instanceId=10, state=active))
        at com.intellij.serviceContainer.ComponentManagerImpl.instantiateClass(ComponentManagerImpl.kt:1022)
        at com.intellij.serviceContainer.ComponentManagerImpl.instantiateClass(ComponentManagerImpl.kt:1039)
        at com.intellij.openapi.options.ConfigurableEP$ClassProducer.createElement(ConfigurableEP.java:438)
        at com.intellij.openapi.options.ConfigurableEP.createConfigurable(ConfigurableEP.java:344)
        at com.intellij.openapi.options.ex.ConfigurableWrapper.createConfigurable(ConfigurableWrapper.java:43)
        at com.intellij.openapi.options.ex.ConfigurableWrapper.getConfigurable(ConfigurableWrapper.java:125)
        at com.intellij.openapi.options.ex.ConfigurableWrapper.cast(ConfigurableWrapper.java:100)
        at com.intellij.openapi.options.ex.ConfigurableWrapper.getDisplayName(ConfigurableWrapper.java:146)
        at com.intellij.ide.util.gotoByName.GotoActionModel.lambda$new$0(GotoActionModel.java:79)
        at com.intellij.openapi.util.NotNullLazyValue$4.compute(NotNullLazyValue.java:120)
        at com.intellij.openapi.util.NotNullLazyValue$4.getValue(NotNullLazyValue.java:105)
        at com.intellij.ide.util.gotoByName.GotoActionModel.getConfigurablesNames(GotoActionModel.java:367)
        at com.intellij.ide.util.gotoByName.GotoActionItemProvider.processOptions(GotoActionItemProvider.java:148)
        at com.intellij.ide.util.gotoByName.GotoActionItemProvider.filterElements(GotoActionItemProvider.java:94)
        at com.intellij.ide.actions.searcheverywhere.ActionSearchEverywhereContributor.lambda$fetchWeightedElements$1(ActionSearchEverywhereContributor.java:99)
        at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$13(CoreProgressManager.java:604)
        at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:679)
        at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:635)
        at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:603)
        at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:60)
        at com.intellij.ide.actions.searcheverywhere.ActionSearchEverywhereContributor.fetchWeightedElements(ActionSearchEverywhereContributor.java:98)
        at com.intellij.ide.actions.searcheverywhere.MixedResultsSearcher$ContributorSearchTask.run(MixedResultsSearcher.java:183)
        at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:227)
        at com.intellij.util.ConcurrencyUtil.lambda$underThreadNameRunnable$3(ConcurrencyUtil.java:215)
        at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:250)
        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:702)
        at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:699)
        at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
        at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:699)
        at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.NullPointerException: Cannot invoke "java.net.URL.toExternalForm()" because "location" is null
        at java.desktop/javax.swing.ImageIcon.<init>(ImageIcon.java:234)
        at io.unthrottled.doki.settings.ThemeSettingsUI.$$$setupUI$$$(ThemeSettingsUI.java)
        at io.unthrottled.doki.settings.ThemeSettingsUI.<init>(ThemeSettingsUI.java:45)
        at com.intellij.serviceContainer.ComponentManagerImpl.instantiateClass(ComponentManagerImpl.kt:981)
```
```
#c.i.o.o.e.ConfigurableWrapper - Can't instantiate configurable for io.unthrottled.doki.settings.ThemeSettingsUI
java.lang.Throwable: Can't instantiate configurable for io.unthrottled.doki.settings.ThemeSettingsUI
        at com.intellij.openapi.diagnostic.Logger.error(Logger.java:202)
        at com.intellij.openapi.options.ex.ConfigurableWrapper.getConfigurable(ConfigurableWrapper.java:127)
        at com.intellij.openapi.options.ex.ConfigurableWrapper.cast(ConfigurableWrapper.java:100)
        at com.intellij.openapi.options.ex.ConfigurableWrapper.getDisplayName(ConfigurableWrapper.java:146)
        at com.intellij.ide.util.gotoByName.GotoActionModel.lambda$new$0(GotoActionModel.java:79)
        at com.intellij.openapi.util.NotNullLazyValue$4.compute(NotNullLazyValue.java:120)
        at com.intellij.openapi.util.NotNullLazyValue$4.getValue(NotNullLazyValue.java:105)
        at com.intellij.ide.util.gotoByName.GotoActionModel.getConfigurablesNames(GotoActionModel.java:367)
        at com.intellij.ide.util.gotoByName.GotoActionItemProvider.processOptions(GotoActionItemProvider.java:148)
        at com.intellij.ide.util.gotoByName.GotoActionItemProvider.filterElements(GotoActionItemProvider.java:94)
        at com.intellij.ide.actions.searcheverywhere.ActionSearchEverywhereContributor.lambda$fetchWeightedElements$1(ActionSearchEverywhereContributor.java:99)
        at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$13(CoreProgressManager.java:604)
        at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:679)
        at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:635)
        at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:603)
        at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:60)
        at com.intellij.ide.actions.searcheverywhere.ActionSearchEverywhereContributor.fetchWeightedElements(ActionSearchEverywhereContributor.java:98)
        at com.intellij.ide.actions.searcheverywhere.MixedResultsSearcher$ContributorSearchTask.run(MixedResultsSearcher.java:183)
        at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:227)
        at com.intellij.util.ConcurrencyUtil.lambda$underThreadNameRunnable$3(ConcurrencyUtil.java:215)
        at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:250)
        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:702)
        at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:699)
        at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
        at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:699)
        at java.base/java.lang.Thread.run(Thread.java:833)
```

--- a/buildSrc/assets/templates/dark.scheme.template.xml
+++ b/buildSrc/assets/templates/dark.scheme.template.xml
@@ -123,6 +123,7 @@
     <option name="ScrollBar.thumbBorderColor" value="%accentColor%5a"/>
     <option name="ScrollBar.thumbColor" value="%accentColor%5a"/>
     <option name="ScrollBar.trackColor" value="$textEditorBackground$"/>
+    <option name="ScrollBar.background" value="$textEditorBackground$"/>
     <option name="TEARLINE_COLOR" value="$lineNumberColor$69"/>
     <option name="TOOLTIP" value="$textEditorBackground$"/>
     <option name="VCS_ANNOTATIONS_COLOR_1" value="$headerColor$"/>

--- a/buildSrc/assets/templates/light.scheme.template.xml
+++ b/buildSrc/assets/templates/light.scheme.template.xml
@@ -121,6 +121,7 @@
     <option name="ScrollBar.thumbBorderColor" value="%accentColor%5a"/>
     <option name="ScrollBar.thumbColor" value="%accentColor%5a"/>
     <option name="ScrollBar.trackColor" value="$textEditorBackground$"/>
+    <option name="ScrollBar.background" value="$textEditorBackground$"/>
     <option name="TEARLINE_COLOR" value="$lineNumberColor$69"/>
     <option name="TOOLTIP" value="$textEditorBackground$"/>
     <option name="VCS_ANNOTATIONS_COLOR_1" value="$headerColor$"/>

--- a/changelog/CHANGELOG.md
+++ b/changelog/CHANGELOG.md
@@ -1,9 +1,10 @@
 Changelog
 ---
 
-# 88.3-1.9.0 [Bug Fixes]
+# 88.3-1.9.1 [Bug Fixes]
 
 - Themed New UI Scrollbars in Rider to be consistent.
+- Fixed the settings window in builds `231.6890.12`+ 
 
 # 88.3-1.9.0 [EXP UI Search Field Fix]
 

--- a/changelog/CHANGELOG.md
+++ b/changelog/CHANGELOG.md
@@ -1,5 +1,10 @@
 Changelog
 ---
+
+# 88.3-1.9.0 [Bug Fixes]
+
+- Themed New UI Scrollbars in Rider to be consistent.
+
 # 88.3-1.9.0 [EXP UI Search Field Fix]
 
 - Vertically aligned the editor search text input on the new UI.

--- a/changelog/RELEASE-NOTES.md
+++ b/changelog/RELEASE-NOTES.md
@@ -4,3 +4,4 @@
 - Vertically aligned the editor search text input on the new UI.
 - Allow custom stickers to remain on IDE for non-Doki themes.
 - Themed New UI Scrollbars in Rider to be consistent.
+- Fixed the settings window in builds `231.6890.12`+ 

--- a/changelog/RELEASE-NOTES.md
+++ b/changelog/RELEASE-NOTES.md
@@ -3,3 +3,4 @@
 - Updated the border color of the `Code Lense` feature.
 - Vertically aligned the editor search text input on the new UI.
 - Allow custom stickers to remain on IDE for non-Doki themes.
+- Themed New UI Scrollbars in Rider to be consistent.

--- a/gradle.properties
+++ b/gradle.properties
@@ -2,7 +2,7 @@
 # -> https://www.jetbrains.org/intellij/sdk/docs/reference_guide/intellij_artifacts.html
 
 pluginGroup=io.unthrottled
-pluginVersion=88.3-1.9.0
+pluginVersion=88.3-1.9.1
 pluginSinceBuild=222
 pluginUntilBuild = 231.*
 
@@ -32,7 +32,7 @@ idePrefix=IU
 idePath=
 #idePath=/Users/alexsimons/Library/Application Support/JetBrains/Toolbox/apps/IDEA-U/ch-0/222.4167.29/IntelliJ IDEA 2022.1 EAP.app/Contents
 #idePath=C:\\Users\\birdm.DESKTOP-FO92PV5\\AppData\\Local\\JetBrains\\Toolbox\\apps\\DataSpell\\ch-0\\213.5744.251
-#idePath = /home/alex/.local/share/JetBrains/Toolbox/apps/AndroidStudio/ch-0/203.7321754
+#idePath=/home/alex/.local/share/JetBrains/Toolbox/apps/AndroidStudio/ch-0/203.7321754
 #idePath=C:\\Users\\birdm.DESKTOP-FO92PV5\\AppData\\Local\\JetBrains\\Toolbox\\apps\\Rider\\ch-0\\212.4416.21
 #idePath=/home/alex/.local/share/JetBrains/Toolbox/apps/IDEA-U/ch-0/202.7660.26
 #idePath=/home/alex/.local/share/JetBrains/Toolbox/apps/PhpStorm/ch-0/193.5233.101

--- a/src/main/java/io/unthrottled/doki/settings/ThemeSettingsUI.form
+++ b/src/main/java/io/unthrottled/doki/settings/ThemeSettingsUI.form
@@ -140,7 +140,6 @@
                                       <grid row="1" column="0" row-span="1" col-span="2" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
                                     </constraints>
                                     <properties>
-                                      <icon value="actions/help.png"/>
                                       <text resource-bundle="messages/MessageBundle" key="settings.general.content.sticker.margin.info"/>
                                     </properties>
                                   </component>
@@ -446,7 +445,6 @@
                               <grid row="1" column="0" row-span="1" col-span="1" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
                             </constraints>
                             <properties>
-                              <icon value="general/warning.png"/>
                               <text resource-bundle="messages/MessageBundle" key="settings.general.weeb.background.editor.info"/>
                             </properties>
                           </component>

--- a/src/main/java/io/unthrottled/doki/settings/ThemeSettingsUI.java
+++ b/src/main/java/io/unthrottled/doki/settings/ThemeSettingsUI.java
@@ -1,5 +1,6 @@
 package io.unthrottled.doki.settings;
 
+import com.intellij.icons.AllIcons;
 import com.intellij.ide.BrowserUtil;
 import com.intellij.ide.DataManager;
 import com.intellij.openapi.application.Application;
@@ -101,6 +102,9 @@ public class ThemeSettingsUI implements SearchableConfigurable, Configurable.NoS
   }
 
   private void initializeAutoCreatedComponents() {
+    warningLabel.setIcon(AllIcons.General.Warning);
+    marginHelp.setIcon(AllIcons.Actions.Help);
+
     chooseImageButton.addActionListener(e -> {
       CustomStickerChooser dialog = new CustomStickerChooser(
         Arrays.stream(ProjectManager.getInstance().getOpenProjects()).findFirst().orElse(

